liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -712,7 +655,7 @@ public static boolean isWhitespace(CharSequence string) {
         if (string == null) {
```

Comment:
Same here.. should we deprecate this entire class and say "USe StringUtils from commons instead"?